### PR TITLE
Styling and UX 

### PR DESCRIPTION
**Pull request by CathrinTruchan**
Hello :)
I have been working a bit more on my styling and the user experience. Thanks for reviewing my changes.

### CODE REVIEW

File: components/FlowCard.js [hunk 1]
```diff
@@ -86,7 +86,16 @@ const StyledButton = styled.button`
 const StyledButtonWrapper = styled.button`
   all: unset;
   position: absolute;
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
```
Comment (philmetscher) [inline components/FlowCard.js:91]: why is here a `border-radius`?

File: pages/flow/[id].js [hunk 2]
```diff
@@ -83,6 +80,12 @@ export default function FlowPage({ asanas, currentFlowDB }) {
     });
   }
 
+  function handleMaxLength(text) {
+    if (text.length > 100) {
+      setDisabledSaveButton(true);
+    } else setDisabledSaveButton(false);
+  }
```
Comment (philmetscher) [inline pages/flow/[id].js:87]: You could shorten this a lot :)
```suggestion
  const handleMaxLength = (text) => setDisabledSaveButton(text.length > 100);
```

File: pages/flow/[id].js [hunk 3]
```diff
@@ -118,21 +121,29 @@ export default function FlowPage({ asanas, currentFlowDB }) {
...
+                The maximum length of the description is 100 characters.
+              </StyledWarning>
+            )}
 
             <MainButton
               type="primary"
-              width="5rem"
+              width="7rem"
               aria-label="Save description"
+              disabled={disabledSaveButton ? true : false}
```
Comment (philmetscher) [inline pages/flow/[id].js:142]: Why you make a statement here. Just use this:
```suggestion
              disabled={disabledSaveButton}
```

File: pages/flow/[id].js [hunk 4]
```diff
@@ -252,10 +260,10 @@ const StyledListItem = styled.li`
   margin: 1.2rem 0.5rem;
 `;
 
-const StyledAddButton = styled.button`
+const StyledRoundButton = styled.button`
   border: none;
   background-color: var(--background-neutral);
-  color: var(--highlight);
+  color: ${(props) => props.color || "var(--highlight)"};
```
Comment (philmetscher) [inline pages/flow/[id].js:266]:
```suggestion
  color: ${({ color }) => color || "var(--highlight)"};
```

File: pages/index.js [hunk 5]
```diff
@@ -112,74 +110,72 @@ export default function Home({ flowsDB }) {
...
+
+        <h3>Let&apos;s flow together</h3>
+
+        <Image
+          src="/images/animation-start-s2.gif"
+          width={300}
+          height={300}
+          alt="yoga"
+          layout="responsive"
+        />
```
Comment (philmetscher) [inline pages/index.js:123]: I think its to big for mobile devices 🤔

**Review by philmetscher — CHANGES_REQUESTED** [review]
some small changes :)

File: components/FlowCard.js (hunk 1 above)
Comment (CathrinTruchan) [reply components/FlowCard.js:91]: Totally useless 😅 I first had a round background and forgot to delete the border-radius. Thanks for the hint

File: pages/flow/[id].js (hunk 2 above)
Comment (CathrinTruchan) [reply pages/flow/[id].js:87]: Ah, nice one :) Thanks!

File: pages/flow/[id].js (hunk 3 above)
Comment (CathrinTruchan) [reply pages/flow/[id].js:142]: Done, thanks :)

File: pages/flow/[id].js (hunk 4 above)
Comment (CathrinTruchan) [reply pages/flow/[id].js:266]: Done, thanks for the hint :)

File: pages/index.js (hunk 5 above)
Comment (CathrinTruchan) [reply pages/index.js:123]: True! I changed it :)

**Comment by CathrinTruchan** [comment]
> some small changes :)

Hi @philmetscher, thanks for your review and the useful feedback. I implemented your changes. Can you have another look? Thanks!

**Review by philmetscher — APPROVED** [review]

**Pull request merged**